src/Magnum/BulletIntegration/MotionState.h

```diff
@@ -43,7 +43,31 @@ namespace Magnum { namespace BulletIntegration {
 
 Encapsulates `btMotionState` as @ref SceneGraph feature.
 
-@todoc Usage...
+# Usage
```

Comment:
This should be `## Usage`, otherwise the header gets way too big in Doxygen output.